integration/tests/cook/test_basic.py

```diff
@@ -1690,7 +1695,8 @@ def test_basic_docker_job(self):
             self.cook_url,
             command='cat /.dockerenv',
             container={'type': 'DOCKER',
-                       'docker': {'image': image}})
+                       'docker': {'image': image}},
+            max_retries=5)
```

Comment:
Let the job retry, so that a single failed instance does not cause the test to fail.